style.css

```diff
@@ -92,11 +91,1192 @@ body {
   left: 24px;
 }
 
-.socialmediaicons {
+.social-media-icons {
   list-style-type: none;
   align-items: flex-start;
   justify-content: space-between;
   margin-left: -38px;
   display: flex;
   width: 150px;
 }
+
+/* NEW FROM HERE */
+.work-grid-container {
+  display: grid;
+  grid-template-rows: auto;
+  row-gap: 10px;
+  padding: 114px 24px;
+}
+
+.work1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  box-sizing: border-box;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
As a [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the repetition of styles. This will make your code more concise. Since every project is similar, uniform classes can serve to style similar portions of the markup.

For example, instead of having to style `work1`, `work2`, `workX`, we could style just `work` and have it affect all.